traits/constants.py

```diff
@@ -136,43 +149,43 @@ class DefaultValue(IntEnum):
     unspecified = -1
 
     #: The default_value of the trait is the default value.
-    constant = 0
+    constant = _CONSTANT_DEFAULT_VALUE
 
     #: The default_value of the trait is Missing.
-    missing = 1
+    missing = _MISSING_DEFAULT_VALUE
 
     #: The object containing the trait is the default value.
-    object = 2
+    object = _OBJECT_DEFAULT_VALUE
 
     #: A new copy of the list specified by default_value is the default value.
-    list_copy = 3
+    list_copy = _LIST_COPY_DEFAULT_VALUE
 
     #: A new copy of the dict specified by default_value is the default value.
-    dict_copy = 4
+    dict_copy = _DICT_COPY_DEFAULT_VALUE
 
     #: A new instance of TraitListObject constructed using the default_value list
     #: is the default value.
-    trait_list_object = 5
+    trait_list_object = _TRAIT_LIST_COPY_DEFAULT_VALUE
```

Comment:
Naming nitpick: we probably want `_TRAIT_LIST_OBJECT_DEFAULT_VALUE` (and similarly for `trait_dict_object`)